src/commands/createStaticWebApp/StaticWebAppCreateStep.ts

```diff
@@ -23,7 +23,7 @@ export class StaticWebAppCreateStep extends AzureWizardExecuteStep<IStaticWebApp
             buildProperties: {
                 appLocation: wizardContext.appLocation,
                 apiLocation: wizardContext.apiLocation,
-                appArtifactLocation: wizardContext.appArtifactLocation
+                appArtifactLocation: wizardContext.outputLocation
```

Comment:
You should add a comment in the code similar to what you put in the PR description